include/boost/winasio/http/basic_http_queue.hpp

```diff
@@ -14,15 +18,15 @@ namespace net = boost::asio;
 namespace winnet = net::windows;
 
 template <typename Executor = net::any_io_executor>
-class basic_http_handle : public winnet::basic_overlapped_handle<Executor> {
+class basic_http_queue : public winnet::basic_overlapped_handle<Executor> {
```

Comment:
This is returned from:
HttpCreateHttpHandle
or HttpCreateRequestQueue
and is handle wrapper.

Maybe it is still good to name it basic_http_handle or basic_http_queue_handle?
The parent class is basic_overlapped_handle.